getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -725,14 +729,14 @@ return early with the ``return`` keyword, but they can't return any value.
           error because if the block is not executed, the function won't have a
           valid value to return.
 
-Referencing Functions
+Referencing functions
 ^^^^^^^^^^^^^^^^^^^^^
 
-Contrary to Python, functions are *not* first class objects in GDScript. This
+Contrary to Python, functions are *not* first-class objects in GDScript. This
 means they cannot be stored in variables, passed as an argument to another
 function or be returned from other functions. This is for performance reasons.
 
-To reference a function by name at runtime, (e.g. to store it in a variable, or
+To reference a function by name at run-time, (e.g. to store it in a variable, or
```

Comment:
Interchangeable, but "runtime" is by far the more common usage, at least according to google trends.

Reply:
I personally got used to write "run-time" to mean the opposite of "compile-time", and write "runtime" in nouns such as "Java runtime environment". Still, I can replace it with "runtime" if this is desired.